frontend/src/components/OdhAppCard.tsx

```diff
@@ -175,27 +163,35 @@ const OdhAppCard: React.FC<OdhAppCardProps> = ({ odhApp }) => {
       data-id={odhApp.metadata.name}
       id={odhApp.metadata.name}
       role="listitem"
-      isHoverable={!disabled}
       className={cardClasses}
       isSelected={selected}
       isSelectable={!disabled}
     >
-      <CardHeader style={{ paddingRight: 0 }}>
-        <CardHeaderMain style={{ maxWidth: '33%', width: '100%' }}>
-          <BrandImage src={odhApp.spec.img} alt={odhApp.spec.displayName} />
-        </CardHeaderMain>
-        <CardActions hasNoOffset>
-          {disabled ? disabledPopover : null}
-          <Dropdown
-            removeFindDomNode
-            onSelect={onOpenKebab}
-            toggle={<KebabToggle onToggle={onToggle} />}
-            isOpen={isOpen}
-            isPlain
-            dropdownItems={dropdownItems}
-            position={'right'}
-          />
-        </CardActions>
+      <CardHeader
+        actions={{
+          actions: (
+            <>
+              {disabled ? disabledPopover : null}
+              <Dropdown
+                onSelect={onOpenKebab}
+                toggle={<KebabToggle onToggle={(e, value: boolean) => onToggle(value)} />}
+                isOpen={isOpen}
+                isPlain
+                dropdownItems={dropdownItems}
+                position={'right'}
+              />
+            </>
+          ),
+          hasNoOffset: true,
+          className: undefined,
+        }}
+        style={{ paddingRight: 0 }}
+      >
+        {
+          <>
+            <BrandImage src={odhApp.spec.img} alt={odhApp.spec.displayName} />
+          </>
+        }
```

Comment:
Should this not just be `<BrandImage ... />`? You're creating a variable block, wrapping a fragment, then rendering a single item 🤔